common/metrics/tags.go

```diff
@@ -31,38 +32,38 @@ type Tag interface {
 	Value() string
 }
 
-type domainTag struct {
+type _domainTag struct {
```

Comment:
We don't use underscore at all in our code base. I don't see this is a good start here to introduce new code style.